vms/platformvm/txs/mempool/mempool.go

```diff
@@ -59,15 +70,18 @@ type Mempool interface {
 	// (both decision and staker) are included into Standard blocks.
 	// HasTxs allow to check for availability of any mempool transaction.
 	HasTxs() bool
+
 	// PeekTxs returns the next txs for Banff blocks
 	// up to maxTxsBytes without removing them from the mempool.
+	// TODO: remove this
 	PeekTxs(maxTxsBytes int) []*txs.Tx
 
-	HasStakerTx() bool
-	// PeekStakerTx returns the next stakerTx without removing it from mempool.
+	// PeekTx returns the next stakerTx without removing it from mempool.
 	// It returns nil if !HasStakerTx().
 	// It's guaranteed that the returned tx, if not nil, is a StakerTx.
-	PeekStakerTx() *txs.Tx
+	PeekTx() *txs.Tx
```

Comment:
If we change the mempool logic and introduce an iterator instead, I believe we should drop these functions.

Reply:
Just removed it. Good catch